test/test_app.rb

```diff
@@ -39,31 +39,29 @@
     assert_match /<pre><code>one\ntwo\nthree\nfour\n<\/code><\/pre>\n/m, last_response.body
   end
 
-  def nfd utf8
```

Comment:
I think this means we can remove `twitter_cldr` as a development dependency from `gollum.gemspec`, too.

Reply:
Oh wow! Thanks for noticing that.